src/rhub/bare_metal/model/host.py

```diff
@@ -0,0 +1,198 @@
+import enum
+import logging
+from typing import Optional
+
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import expression
+
+from rhub.api import db
+from rhub.api.utils import ModelMixin, TimestampMixin
+from rhub.bare_metal.model.common import (
+    BareMetalArch,
+    BareMetalBootType,
+    IronicNode,
+    _BM_TABLE_NAME_HANDLER,
+    _BM_TABLE_NAME_HOST,
+    _BM_TABLE_NAME_HOST_DRAC,
+    _BM_TABLE_NAME_HOST_REDFISH,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class BareMetalHostStatus(str, enum.Enum):
+    AVAILABLE = "available"
+    ENROLLING = "enrolling"
+    FAILED_ENROLLING = "failed_enrolling"
+    MAINTENANCE = "maintenance"
+    RESERVED = "reserved"
+    UNAVAILABLE = "unavailable"
+
+
+class BareMetalHostVendor(str, enum.Enum):
```

Comment:
I don't think 'HostVendor' is appropriately named.  Perhaps HostHWType would be better.